src/Core/src/Platform/Windows/NavigationViewExtensions.cs

```diff
@@ -270,5 +272,22 @@ public static void UpdateFlyoutWidth(this MauiNavigationView navigationView, IFl
 			// At some point this Template Setting is going to show up with a bump to winui
 			//handler.PlatformView.OpenPaneLength = handler.PlatformView.TemplateSettings.OpenPaneWidth;
 		}
+
+		public static async Task UpdateFlyoutIconAsync(this AnimatedIcon platformView, IImageSource? imageSource, IImageSourceServiceProvider? provider)
+		{
+			if (platformView is null)
+				return;			
+
+			if (provider is not null && imageSource is not null)
```

Comment:
If the user hasn't set the `icon` can we return it back to the font glyph?

https://github.com/microsoft/microsoft-ui-xaml/blob/a7183df20367bc0e2b8c825430597a5c1e6871b6/dev/NavigationView/NavigationView_rs1_themeresources.xaml#L389-L391